app/connectors/BalanceRequestConnector.scala

```diff
@@ -63,7 +62,7 @@ trait BalanceRequestConnector {
 class BalanceRequestConnectorImpl @Inject() (
   appConfig: AppConfig,
   http: HttpClient,
-  val metrics: Metrics
+  val metrics: com.codahale.metrics.MetricRegistry
```

Comment:
Please move ```com.codahale.metrics``` at top. should be part of import. Please do it for all other places as well

Reply:
also With play ```8.4.0``` we have ```Metrics``` so we can use that one ```import uk.gov.hmrc.play.bootstrap.metrics.Metrics```. instead of ```com.codahale.metrics.MetricRegistry``` and then it is going to be same just only package level change just like we did for ```akka``` to ```pekko```